cilium/cmd/encrypt_status.go

```
@@ -37,6 +59,43 @@ func init() {
 	command.AddJSONOutput(encryptStatusCmd)
 }
 
+func getIpsecmode() string {
+	if option.Config.EnableIPSec {
+		return EncryptionModeIPsec
+	} else if option.Config.EnableWireguard {
+		return EncryptionModeWireguard
+	}
+	return EncryptionModeDisabled
+}
+
+func getXfrmStats() (int, map[string]int) {
+	fs, _ := procfs.NewDefaultFS()
+	stats, _ := fs.NewXfrmStat()
```

This might silently fail. I think it'd be nice to a least log errors.